src/lib/db/api-token-store.ts

```diff
@@ -173,7 +173,10 @@ export class ApiTokenStore implements IApiTokenStore {
     }
 
     async get(key: string): Promise<IApiToken> {
-        const row = await this.makeTokenProjectQuery().where('secret', key);
+        const row = await this.makeTokenProjectQuery().where(
+            'tokens.secret',
+            key,
+        );
```

Comment:
tests started failing here due to ambiguous column names, so I took the liberty of adding the table name we use in makeTokenProjectQuery as a prefix.